### ISSUE TITLE
PidGraph for Linux looks bad

### ISSUE THREAD

**Issue by eunovm** [17451k/clade#63]
For instance, look at:
[pid_graph.dot.pdf](https://github.com/17451k/clade/files/2896627/pid_graph.dot.pdf)

I hope that this is the issue just with representation pid graphs as a picture.

**Comment by 17451k** [comment]
I will need more context for this issue.

**Comment by eunovm** [comment]
I tried to process one module of Linux 3.14.79 (in addition, I changes configuration for "linux_kernel" to print PDFs):
```
make defconfig
clade-intercept make -j16 net/netfilter/xt_nat.ko
clade-callgraph -p linux_kernel cmds.txt
```

**Comment by 17451k** [comment]
Fixed in 39b74374a1aa57428d73743e77951234a8f720d4.
